src/styles/globals/base.scss

```diff
@@ -22,3 +22,18 @@
 select {
   appearance: none;
 }
+
+
+.mySwiper{
```

Comment:
Maybe this is my lack of NextJS knowledge, but why is this padding here and referred to by `className="mySwiper"` instead of adding the padding to the className?

Reply:
The 'mySwiper' classname is a predefined classname by the package if I'm not wrong. It has all the predefined styles.

Reply:
Oh I think I know what you mean, the padding is overwritten by the initial styles defined in the mySwiper if put inside the classname tag, so putting it in the css file with the !important, gives is priority.